internal/runtime/hcsv2/uvm.go

```diff
@@ -354,6 +357,15 @@ func modifyMappedVPMemDevice(ctx context.Context, rt prot.ModifyRequestType, vpd
 	}
 }
 
+func modifyMappedVPCIDevice(ctx context.Context, rt prot.ModifyRequestType, vpciDev *prot.MappedVPCIDeviceV2) error {
+	switch rt {
```

Comment:
Do we need to handle the remove case here?

Reply:
In the gpu case goes, I don't think so. AFAIK nvidia-container-cli doesn't support removing a gpu from a container, so the only removal that needs to be done is of the vpci device on the uvm.